### ISSUE TITLE
Laravel 5

### ISSUE THREAD

**Issue by TheTechnoMan** [Chumper/Datatable#232]
Hey,

any plans to make it compatible with Laravel 5?

Thanks!

**Comment by ProtoTempus** [comment]
+1

**Comment by shellprog** [comment]
+1 L5

**Comment by mouhsinelonly** [comment]
+1

**Comment by Chumper** [comment]
will look into it, now that l5 is out.
I will also look to update to 1.10 and rewrite some parts

**Comment by TheTechnoMan** [comment]
Awesome. Do you have any eta?

PS: Vielen Dank :+1:

**Comment by Chumper** [comment]
No ETA at the moment, sorry.
I need to test and lay out some foundations before i can start.
I want to reduce the work i need to put into both engines, best would be to just maintain one general engine but i need to think about how to achieve that first.

I also want to drop the frontend part as to many people think the plugin should also handle the client side of datatable and get confused.

Another aspect is that i want to add some kind of easy to extend interface so we can stuff something like live editor or custom search into it.

**Comment by TheTechnoMan** [comment]
I see, makes sense. What do you mean with "drop the frontend part"? You mean the feature that it automatically generates the code (renders it into the actual table) and you just have to output it at the desired position? If so, in my opinion you should not remove that as it takes away the pain with manually dealing with the datatables, that is something that i personally really was happy about.

Thanks!

**Comment by Chumper** [comment]
Yeah, i mean the `table()` part, a lot of issues are about how to set the options in the right way and generaly on how to style the html table to their needs. 
That means, i also need to know about most frontend parts of datatable to help with the issues.

I worked with yii before and i want to have something like the CGridView (http://www.yiiframework.com/doc/api/1.1/CGridView) for laravel with datatable as frontend engine.

Maybe it is a good idea to copy the concept of `dataProviders` to achieve a lot of flexibility.

**Comment by TheTechnoMan** [comment]
I understand your point. I would simply mention that you can't provide support for the front end part as there are too many specific parameters to know about. Let's be honest, time is money. If someone needs additional help, they might contact you for some paid freelance work.

For me personally the `table()` part worked really well (with Bootstrap) and gave me all features that I needed, which basically only was the fact that I need to add some buttons to the table depending on the fetch database data (basically a edit button for the corresponding user).

The `dataProviders` concept looks pretty straightforward though.

I do personally just hope that you come up with a support for L5 soon as I really need it to continue the work on the current project (otherwise I could do all manually and switch it out later, but thats not really the way to go).

Thank you very much, Nils! :)

**Comment by BartReymenants** [comment]
+1 L5

**Comment by jandever** [comment]
+1 L5

**Comment by keithmclaughlin** [comment]
+1 L5

**Comment by erenilhan** [comment]
+1 L5

**Comment by TheTechnoMan** [comment]
@Chumper I really don't want to bug you, but do you have any updates?

Thanks!

**Comment by webcrazy** [comment]
+1 L5

**Comment by WiR3D-kNiGhT** [comment]
@TheTechnoMan this is OSS, it will be done when its done. And although you and I and everyone else that +1'd this wants it, we shouldn't bother him, or ask for eta's.

**Comment by TheTechnoMan** [comment]
@WiR3D-kNiGhT I don't see anything wrong with asking if there are some news as long as you don't do it every day ;)

**Comment by Ben-Speakman** [comment]
+1

**Comment by Chumper** [comment]
I currently designed the new i want to use for the new version, there are still some problems to solve but i definetly go forward :)

**Comment by jayeshmagare** [comment]
For those who are looking for step by step guidance in using Datatable with Larave5 , http://jayeshmagare.blogspot.com/2015/02/using-chumper-datatable-in-laravel-5.html

**Comment by TheTechnoMan** [comment]
@Chumper Awesome, looking forward to it. :)

@jayeshmagare Thanks, but I think it is better to just wait instead of modifying core code which will get overwritten with the next update and therefore break things for anyone who modified it.

**Comment by Ben-Speakman** [comment]
There is also a working L5 fork here - https://github.com/bart/Datatable

**Comment by Thatdoorsajar** [comment]
+1

**Comment by carlituxman** [comment]
+1

any estimation?

**Comment by TheTechnoMan** [comment]
Looking really forward to it as well. Hows it going?

Thanks!

**Comment by Chumper** [comment]
Still working on it. I would suggest to use any method provided in the comments. If there are working versions I can merge them in so you have something working atm.

**Comment by carlituxman** [comment]
I've tried @jayeshmagare solution. and works form me while I wait for version 3

**Comment by carlituxman** [comment]
the solution by @jayeshmagare works with some changes:

step 4:
'table_view' => 'Chumper::template',
'script_view' => 'Chumper::javascript',

and at template.blade.php
at last lines:

@if (!$noScript)
@include(Config::get('chumper_datatable.table.script_view'), array('id' => $id, 'options' => $options, 'callbacks' => $callbacks))
@endif

and to render view use {!! ..... !!}}

**Comment by TheTechnoMan** [comment]
Not to bug you, but do you have any update? :(

**Comment by kylescousin** [comment]
Hey Chumper,

Any updates? Please make this compatible with L5.
Lots of people relying on your skills! :)

**Comment by fabioi** [comment]
+1 L5

**Comment by fabioi** [comment]
+1

**Comment by liptonsparkling** [comment]
+1 l5

**Comment by od3n** [comment]
+1 L5

**Comment by scottconnerly** [comment]
+1 L5 :)

**Comment by timgws** [comment]
# Hello everyone!

There is a branch (`develop`) that has support for Laravel 5. For the people that want Laravel 5 support, please try using the `develop` branch, testing it and if any issues come up across the way, open a *new separate bug* for that problem.

This "+1" business does not help anyone, we all know that this is a feature that is in strong demand.

Please stop. Thanks :)

**Comment by ibnusulaiman83** [comment]
+1
